### PR TITLE
[APAM-2]Replace deprecated API and update dependency

### PR DESCRIPTION
**Pull request by passyruan**
Replace deprecated API and update dependency

### CODE REVIEW

File: GUIDE.md [hunk 1]
```diff
@@ -126,14 +127,9 @@ Next Step:
...
-At first, add below code in your host Activity or Fragment, implement Activity#onActivityResult and handle the result.
+At first, call the initialization method of AirwallexStarter in your Application.
 ```kotlin
-    override fun onActivityResult(requestCode: Int, resultCode: Int, data: Intent?) {
-        super.onActivityResult(requestCode, resultCode, data)
-        
-        // You must call this method on `onActivityResult`
-        AirwallexStarter.handlePaymentData(requestCode, resultCode, data)
-    }
+    AirwallexStarter.initialize(mApplication)
```
Comment (aw-hector) [inline GUIDE.md:132]: Do we still need to do this since mApplication is already register during `Airwallex.initialize`?

File: sonar.gradle [hunk 2]
```diff
@@ -88,6 +88,7 @@ sonarqube {
                 '**/*ViewModelFactory.*',
                 '**/*ScreenEvent.*',
                 '**/*ActivityLaunch.*',
```
Comment (aw-hector) [inline sonar.gradle:90]: This might not be needed anymore

File: ui-core/src/main/java/com/airwallex/android/ui/AirwallexActivityLaunch.kt [hunk 3]
```diff
@@ -13,6 +18,59 @@ abstract class AirwallexActivityLaunch<TargetActivity : Activity, ArgsType : Air
     private val requestCode: Int
 ) {
 
+    companion object {
+        private var inited = false
+        private val resultLauncherMap = HashMap<Activity, ActivityResultLauncher<Intent>>()
+        private val resultCallbackMap = HashMap<Activity, AirwallexActivityLaunchResultCallback>()
+
+        fun initialize(application: Application) {
+            if (inited) return
```
Comment (aw-hector) [inline ui-core/src/main/java/com/airwallex/android/ui/AirwallexActivityLaunch.kt:27]: Naming `isInitialized` is better?

File: card/src/main/java/com/airwallex/android/card/CardComponent.kt [hunk 4]
```diff
@@ -25,6 +27,10 @@ class CardComponent : ActionComponent {
     private var dccCallback: DccCallback? = null
     private var listener: Airwallex.PaymentResultListener? = null
 
+    override fun init(application: Application) {
```
Comment (aw-hector) [inline card/src/main/java/com/airwallex/android/card/CardComponent.kt:30]: Can we call it `initialize` for consistency? Init feels like it's system `init {}`

File: GUIDE-zh.md [hunk 5]
```diff
@@ -274,20 +260,70 @@ val paymentSession = AirwallexPaymentSession.Builder(
...
+            .setSupportComponentProviders(
+                listOf(
+                    CardComponent.PROVIDER,
+                    WeChatComponent.PROVIDER,
+                    RedirectComponent.PROVIDER,
+                    GooglePayComponent.PROVIDER
+                )
+            )
+            .build(),
+        ExampleClientSecretProvider()           // If you need to support recurring, you must to support your custom ClientSecretProvider
```
Comment (aw-hector) [inline GUIDE-zh.md:304]: `you must to support` -> `you must provide`, could you please also correct everywhere else this sentence is used?

Comment (passyruan) [reply GUIDE-zh.md:304]: ok

File: GUIDE-zh.md [hunk 6]
```diff
@@ -274,20 +260,70 @@ val paymentSession = AirwallexPaymentSession.Builder(
...
+
+### 配置低层API SDK
+设置SDK后，需要使用一些参数来配置SDK。 在使用Airwallex SDK confirm PaymentIntent并完成付款之前，您应在自己的服务器中创建PaymentIntent，以确保在自己的系统中维护信息
+#### 初始化SDK
+
+我们提供了一些可用于调试SDK的参数，你可以在Application中调用
 ```kotlin
-val session = buildSession(paymentIntent, customerId)
-val airwallex = Airwallex(this@PaymentCartFragment)
+     AirwallexStarter.initialize(
```
[the remark this answers is not in the document]
Comment (passyruan) [reply GUIDE-zh.md:290]: yes, I will fix it

File: card/src/main/java/com/airwallex/android/card/CardComponent.kt (hunk 4 above)
Comment (passyruan) [reply card/src/main/java/com/airwallex/android/card/CardComponent.kt:30]: ok

File: ui-core/src/main/java/com/airwallex/android/ui/AirwallexActivityLaunch.kt (hunk 3 above)
Comment (passyruan) [reply ui-core/src/main/java/com/airwallex/android/ui/AirwallexActivityLaunch.kt:27]: ok

File: GUIDE-zh.md [hunk 7]
```diff
@@ -19,18 +19,17 @@ Airwallex Android SDK是一种灵活的工具，可让您将付款方式集成
...
-    * [配置SDK](#配置SDK)
-        * [初始化SDK](#初始化SDK)
-        * [创建PaymentIntent](#创建PaymentIntent)
-* [UI集成](#UI集成)
+* [UI集成](#ui集成)
+    * [UI集成添加依赖](#UI集成添加依赖)
+    * [UI集成配置SDK](#UI集成配置SDK)
     * [Edit Shipping Info](#edit-shipping-info)
     * [Use the entire Native UI in one flow](#use-the-entire-native-ui-in-one-flow)
     * [Set up Google Pay](#set-up-google-pay)
```
Comment (aw-hector) [inline GUIDE-zh.md:27]: This now needs to be shared across both UI and low-level integration.

File: GUIDE-zh.md [hunk 8]
```diff
@@ -274,20 +260,70 @@ val paymentSession = AirwallexPaymentSession.Builder(
...
+在项目的根目录下，打开`build.gradle`，并添加以下内容：
+
+```groovy
+    dependencies {
+        // It's required
+        implementation 'io.github.airwallex:payment-components-core:4.5.0'
+        
+        // Select the payment method you want to support.
+        implementation 'io.github.airwallex:payment-card:4.5.0'
+        implementation 'io.github.airwallex:payment-redirect:4.5.0'
```
Comment (aw-hector) [inline GUIDE-zh.md:278]: redirect & wechat is not supported yet.

File: GUIDE-zh.md [hunk 9]
```diff
@@ -274,20 +260,70 @@ val paymentSession = AirwallexPaymentSession.Builder(
...
+                    GooglePayComponent.PROVIDER
+                )
+            )
+            .build(),
+        ExampleClientSecretProvider()           // If you need to support recurring, you must to support your custom ClientSecretProvider
+    )
 ```
 
-### 步骤二：在你的Activity或Fragment中实现Activity#onActivityResult
+#### 创建PaymentIntent
```
Comment (aw-hector) [inline GUIDE-zh.md:308]: Can refer to the same process in UI integration.

**Review by aw-hector — APPROVED** [review]
Nice work! 🚀

**Pull request merged**